app/views/images/crop.html.erb

```diff
@@ -25,7 +25,7 @@ content_for :title,t(title_key, title: @edition.title_or_fallback)
     src: url_for(@image_revision.blob),
     no_js_src: url_for(@image_revision.crop_variant('960x640')),
     exact_dimensions: @image_revision.at_exact_dimensions?,
-    alt_text: @image_revision.alt_text,
+    alt_text: @image_revision.alt_text || t("images.edit.alt_text_placeholder"),
```

Comment:
I'm not sure this fits the brief of:

> We need to provide the user with a default/placeholder alt text in the markup for freshly uploaded images. This is not to be stored anywhere - only presented until the alt text is provided by the user.

If the user doesn't make any changes to the alt text, it would now by default be persisted as "Your uploaded image". I would interpret this requirement as being the `placeholder` attribute rather than the `value` of the input element.

Reply:
I don't think I follow this. How would a user end up with "Your uploaded image" be persisted? We're only using this text when rendering the `<img>` tags - not in the forms.

Reply:
Ahhh that makes sense.